### PR TITLE
 Disregard non problem activities for PointsEarned policy

### PR DESCRIPTION
**Pull request by alexbojko**
### [PROBLEM]
is described in ticket [AD-81](https://vpalresearch.atlassian.net/browse/AD-81)

### [SOLUTION]
exclude activities with `is_problem=True` in `BaseGradingPolicy._get_points_earned_trials_count`

### CODE REVIEW

**Comment by alexbojko** [comment]
@kunanit what do you think about this change?

**Comment by kunanit** [comment]
I think this looks good as a base policy. For those activities that have is_problem=False, is the score always 'null'?

**Comment by alexbojko** [comment]
@kunanit I answered to your question in ticket  [AD-81](https://vpalresearch.atlassian.net/browse/AD-81)

File: bridge_adaptivity/config/settings/base.py [hunk 1]
```diff
@@ -139,6 +139,7 @@
 CELERY_TASK_SERIALIZER = 'json'
 CELERY_RESULT_SERIALIZER = 'json'
 
+# This settings is related to module/egines and declare gradable problems
```
Comment (idegtiarov) [inline bridge_adaptivity/config/settings/base.py]: typo s/settings is related/settings are related/

File: bridge_adaptivity/module/tests/test_grading_policy.py [hunk 2]
```diff
@@ -82,14 +82,18 @@ def test_grading_policy_has_name(self, test_cls, public_name):
         self.assertEqual(getattr(test_cls, 'public_name'), public_name)
 
     @mock.patch('module.policies.base.BaseGradingPolicy._get_points_earned_trials_count')
+    @mock.patch('module.policies.policy_points_earned.PointsEarnedGradingPolicy._get_points_earned_trials_count')
```
Comment (idegtiarov) [inline bridge_adaptivity/module/tests/test_grading_policy.py]: Please move `patch` decorators directly under the test function for explicit usage of the function's parameters

**Pull request merged**